src/drivers/intel/cavs/ipc.c

```diff
@@ -23,6 +23,35 @@
 #include <stddef.h>
 #include <stdint.h>
 
+#if CAVS_VERSION >= CAVS_VERSION_1_8
```

Comment:
what if we want to use this for GLK low-power stuff?

Reply:
No issue with that. However, if we map TDR,TDD registers available in 1.8+ to T,TE in 1.5, there are only 30 bits available in TE which limits the payload size field to 6bits which is OK.